README.md

```diff
@@ -11,6 +11,7 @@
 - [Overview](#overview)
 - [Getting Started](#getting-started)
 - [Learning to code with the Java Koans for the FRC](#learning-to-code-with-the-java-koans-for-the-frc)
+- [Using Browser Based IDE](#using-browser-based-ide)
```

Comment:
Would it make sense to have "Getting started on your computer" and "Getting started in your browser" one after the other?

Reply:
That is a good idea.  Hopefully this works for you.